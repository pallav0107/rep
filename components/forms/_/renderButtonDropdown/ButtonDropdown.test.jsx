import React from 'react';
import { shallow } from 'enzyme';
import { withTheme } from '../../../../../utils/jest/TestUtils';
import "../../../../setupTests"

describe('ButtonDropdown', () => {
  it('should render', () => {
    const component = withTheme(<renderButtonDropdown meta={['touched', 'error']} />);
    const wrapper = shallow(component);
    expect(wrapper).toMatchSnapshot();
  });
});
