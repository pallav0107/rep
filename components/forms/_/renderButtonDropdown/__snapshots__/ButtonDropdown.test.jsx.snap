// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonDropdown should render 1`] = `
<renderButtonDropdown
  meta={
    Array [
      "touched",
      "error",
    ]
  }
/>
`;
